pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers = [
 python = "^3.8"
 fastavro = "^1.7.3"
 jsonschema = "^4.17.3"
-httpx = "^0.24"
```

Comment:
I think it is not a good idea as sometimes there are breaking changes. We should try to keep it up to date as much as possible and make sure that it is working. Could you updated to `^0.26.0"?

Reply:
Done ! I also did `poetry lock --no-update` and added the poetry.lock to be sync with the update.